sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/input.py

```diff
@@ -76,7 +76,7 @@ class Input(_InputOutputBase):  # pylint: disable=too-many-instance-attributes
     def __init__(
         self,
         *,
-        type: str,
+        type: Literal["uri_folder"] = "uri_folder",
```

Comment:
There's already an overload below for "uri_folder" - so maybe this overload is just superfluous and should be removed?